src/lib/controllers/ui-controller.js

```diff
@@ -1802,28 +1804,41 @@ export default class UIController {
 
   registerGetSelectedText (listenerName, selector) {
     let ev
+    let customEv
     if (this.platform.isMobile) {
-      ev = LongTap
+      switch (this.options.textQueryTriggerMobile) {
```

Comment:
may be we should do here some normalized method (to lowerCase, remove spaces or hyphen) and remove these several longTap switchers?